docs/source/roles/system_manager/manage_users.md

```diff
@@ -137,6 +137,14 @@ The `DC1` is the source of truth for user details. If these details need to be c
     - Click on `Users` under `Manage` and search for the user
     - Confirm the user is no longer present
 
+### {{x}} Automatically deleting all usassigned users
+
+In some situations, such as at the end of a project after an SRE has been torn down, you may want to remove all users from the SHM who are not assigned to the security group of any remaining attached SREs.
```

Comment:
This shouldn't be "in some situations", it should be a regular thing that we run every month.

Reply:
As discussed, that will be added to TRESA docs, not here